css/properties/max-height.json

```diff
@@ -53,16 +53,15 @@
             "deprecated": false
           }
         },
-        "sizing_values": {
+        "max-content": {
```

Comment:
Has this file been alphabetized or is this a name change?

Reply:
I added the features to the file.

Reply:
@jpmedley these features were decomposed (i.e., `sizing_values` was a combination of `fit-content`, `max-content`, and `min-content`). I suggest taking a look at the split diff instead of the unified diff when you see something like this